sklearn/model_selection/tests/test_search.py

```diff
@@ -2521,8 +2521,7 @@ def test_search_with_2d_array():
     data_target = [0, 0, 1, 0, 1]
     random_search.fit(data_train, data_target)
     result = random_search.cv_results_["param_vect__ngram_range"]
-    expected_data = np.empty(3, dtype=object)
-    expected_data[:] = [(1, 2), (1, 2), (1, 1)]
+    expected_data = np.array([[1, 2], [1, 2], [1, 1]])
```

Comment:
also looks better / more desirable to have a 2D int64 array if possible, rather than an object array of tuples?

Reply:
Yeah people would get a 2-element array instead of a 2-element tuple:
```python
In [18]: a = np.array([[1, 2], [1, 2], [1, 1]])

In [19]: b = np.empty(3, dtype=object)

In [20]: b[:] =  [(1, 2), (1, 2), (1, 1)]

In [21]: a[0]
Out[21]: array([1, 2])

In [22]: b[0]
Out[22]: (1, 2)
```
I'll see if it can be preserved